src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -177,7 +178,7 @@ private IEnumerable<string> ExtractProjectFolders(ProjectAnalyzerResult projectA
             var folders = new List<string>();
             var projectDirectory = _fileSystem.Path.GetDirectoryName(projectFilePath);
             folders.Add(projectDirectory);
-            if (!fullFramework)
+           // if (!fullFramework)
```

Comment:
What is the point of this if? I disabled it while testing for compatibility with NFluent on Net 4.5 (otherwise, the project is seen as empty by Stryker).
Can you please give the reason why Stryker disregards share project for full framework? otherwise I suggest we get rid of this condition altogether.

Reply:
At some point when shared projects were added we thought only dotnet core had shared projects. I guess we forgot to remove this check when we realized this isn't true. The if should be removed